### PR TITLE
add copy feature using titles

### LINKED ISSUE #352: Ability to Copy Request or Response Payloads
I would love to quickly copy the payloads so I can then use them for testing.  Something similar to a copy to clipboard button next to the payload would be awesome.

### PR DESCRIPTION
**Pull request by AnkRaiza**
Fixes #352 

Changes in this pull request include:
-add copy feature to interaction page

screen shot 2017-06-21 at 4 09 03 pm

### CODE REVIEW

File: src/components/Interaction.tsx [hunk 1]
```diff
@@ -98,6 +118,40 @@ export class Interaction extends React.Component<InteractionProps, InteractionSt
         return key;
     }
 
+    handleCopyRequestPayload: (e: any) => void;
+    handleCopyResponsePayload: (e: any) => void;
+    handleCopyPayload(e: any) {
+        const payload = e.target.getAttribute("data-type") === "request" ? this.props.request : this.props.response;
+        const textArea = document.createElement("textarea");
```
Comment (chris-ramon) [inline src/components/Interaction.tsx]: Perhaps we should prefer object literal here.
```typescript
textArea.style = {
  position: "fixed",
  top: 0,
  left: 0,
  // ...
};
```

**Comment by chris-ramon** [comment]
also perhaps, we want to use another icon?, such as:
Github:
image

Font Awesome:
image

/cc: @jkelvie

File: src/components/Interaction.tsx (hunk 1 above)
Comment (AnkRaiza) [inline src/components/Interaction.tsx]: style is a readonly property in this context, so i can't actually replace it with object literal, i can only set their properties values

Comment (chris-ramon) [inline src/components/Interaction.tsx]: ok cool, didn't know that :+1:

**Pull request merged**